packages/now-cli/src/util/input/input-project.ts

```diff
@@ -93,13 +101,22 @@ export default async function inputProject(
     });
     newProjectName = answers.newProjectName as string;
 
+    if (!newProjectName) {
+      output.print(`${chalk.red('Error!')} Project name can not be empty\n`);
```

Comment:
```suggestion
      output.print(`${chalk.red('Error!')} Project name cannot be empty\n`);
```

Reply:
TIL https://www.dailywritingtips.com/cannot-or-can-not/

Reply:
> The experts at AskOxford seem to prefer cannot

If the experts say it!

Reply:
Thanks for the suggestion @AndyBitz 😎